core/src/main/java/apoc/load/LoadArrow.java

```diff
@@ -109,7 +109,7 @@ public Stream<MapResult> stream(
     }
 
     @Procedure(name = "apoc.load.arrow")
-    @Description("apoc.load.arrow(fileName, config) - imports nodes and relationships from the provided file")
+    @Description("Imports nodes and relationships from the provided arrow file.")
```

Comment:
The docs PR did not have the word 'arrow' there if you want to be consistent, but on the other hand I think it is good to include it...

Reply:
Ah hehe I'll make a PR to docs for this as I think it is good :)